cmd/snap/cmd_snap_op.go

```diff
@@ -1002,6 +1004,7 @@ func init() {
 		}), nil)
 	addCommand("refresh", shortRefreshHelp, longRefreshHelp, func() flags.Commander { return &cmdRefresh{} },
 		waitDescs.also(channelDescs).also(modeDescs).also(map[string]string{
+			"amend":             i18n.G("Amend snap to match the store version"),
```

Comment:
I think the word "amend" is too complex and will not be understood by random user. How about "replace"?

Reply:
Reading the PR title my initial thought was "something to modify an installed snap( from a local build?)". But that's not what it is.
For comparison there's `git commit --amend` to update an existing commit with local changes. And outside of software you can "amend law" which tends to mean a new revision of existing documents.

To my mind "replace" would make more sense because what you install can be completely different - we don't know if it's even the same snap, safe for the name.